src/decorator_trait.jl

```diff
@@ -330,20 +330,24 @@ function is_point(
     kwargs...,
 )
     # to be safe check_size first
+    try
+        pt = is_point(get_embedding(M, p), embed(M, p), te; kwargs...)
+        !pt && return false # no error thrown (deactivated) but returned false -> return false
+    catch e
+        if e isa DomainError || e isa AbstractManifoldDomainError
```

Comment:
If we get a different error we probably want to rethrow it? If I see correctly the current code makes them ignored.

Reply:
oh yes, the original code had the throw outside (hence the e that just was reencapsulated), but you're right we should re throw them.